x-pack/plugins/remote_clusters/__jest__/client_integration/remote_clusters_list.test.js

```diff
@@ -241,8 +242,10 @@ describe('<RemoteClusterList />', () => {
         actions.clickBulkDeleteButton();
```

Comment:
Not related to the changes in this PR but I am surprised that it does not complain to update the state outside an `act()` call for those 2 actions

Reply:
I'm going to leave this as is for now, since it is unrelated to the current changes.